code/modules/research/designs/autolathe_designs.dm

```diff
@@ -195,6 +195,14 @@
 	build_path = /obj/item/floor_painter
 	category = list("initial", "Miscellaneous")
 
+/datum/design/airlock_painter
+	name = "Airlock painter"
+	id = "airlock_painter"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 3000, MAT_GLASS = 1000)
```

Comment:
I'm not familiar with this code, but is both this and `materials` var in `/obj/item/airlock_painter` necessary? How are the two different, and what happens if you forget to specify one of them?

Reply:
The materials list here is how much it costs to print said item. The materials list in the airlock painter file determines the amount of material that gets added whenever you recycle it into an autolathe, or anything else that will accept it. I think it also determines what's dropped on the floor if the item is destroyed, but not sure on that.

If you forgot to specify them, making these from an autolathe would be free and recycling them would give no materials.